docs/jwst/photom/main.rst

```diff
@@ -68,36 +77,37 @@ The process of attaching the pixel
 area data also populates the keywords PIXAR_SR and PIXAR_A2 in the primary
 header of the science product, which give the average pixel area in units of
 steradians and square arcseconds, respectively.
-Both the photom and pixel area reference files contain the average pixel
-area values in their primary headers. The photom step copies the values from
-the pixel area reference file to populate the PIXAR_SR and PIXAR_A2 keywords
-in the science data. It will issue a warning if the values of those keywords
-in the two reference files differ by more than 0.1%.
+The photom step copies the values from the pixel area reference file to
+populate the PIXAR_SR and PIXAR_A2 keywords in the science data.  The values
+are taken either from the primary header or, for NIRSpec data, from a binary
+table extension.
+Note that while both the photom and pixel area reference files contain these
+keywords, the values are taken only from the pixel area reference file.
 
 NIRSpec IFU
 -----------
 
 The photom step uses the same type of tabular reference file for NIRSpec IFU
 exposures as discussed above for other modes, where there is a single table
 row that corresponds to a given exposure's filter and grating settings. It
-retreives the scalar conversion constant, as well as the 1-D wavelength and
+retrieves the scalar conversion constant, as well as the 1-D wavelength and
 relative response arrays, from that row. It also loads the IFU pixel area
 data from the pixel area reference file.
 
 It then uses the scalar conversion constant, the 1-D wavelength and relative
 response, and pixel area data to compute a 2-D sensitivity map (pixel-by-pixel)
 for the entire science image. The 2-D SCI and ERR arrays in the science
-exposure are divided by the 2D sensitivity map, which converts the science
+exposure are multiplied by the 2D sensitivity map, which converts the science
 pixels from units of DN/sec to mJy/arcsec\ :sup:`2`\ .
-Variance arrays are divided by the square of the conversion factors.
+Variance arrays are multiplied by the square of the conversion factors.
 
 MIRI MRS
 --------
 
 For the MIRI MRS mode, the photom reference file contains 2-D arrays of sensitivity
 factors and pixel sizes that are loaded into the step. As with NIRSpec IFU, the
 sensitivity and pixel size data are used to compute a 2-D sensitivity map
-(pixel-by-pixel) for the entire science image. This is divided into both
+(pixel-by-pixel) for the entire science image. This is multiplied into both
 the SCI and ERR arrays of the science exposure, which converts the pixel values
 from units of DN/sec to mJy/arcsec\ :sup:`2`\ .
```

Comment:
Same question about resulting units. I'm guessing everything will now be in MJy/sr.